src/api/httpClient.ts

```diff
@@ -1,33 +1,33 @@
 import { Axios, AxiosError, AxiosResponse } from 'axios';
 import { ErrorResponse, ResponseData } from './response';
 
-export interface Header {
+export interface AxiosHeader {
```

Comment:
It is just a header. Not related to underlying implementation

Reply:
Was AxiosHeader in other project so thats why I renamed it. Reverted this change